### PR TITLE
Fix typo "priori" -> "prior"

### PR DESCRIPTION
**Pull request by satwanjyu**
_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any): Fix typo in `src/resources/inside-flutter.md:589`

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: src/resources/inside-flutter.md [hunk 1]
```diff
@@ -586,7 +586,7 @@ Some interpolatable objects are defined by class hierarchies. For example,
 shapes are represented by the `ShapeBorder` interface, and there exists a
 variety of shapes, including `BeveledRectangleBorder`, `BoxBorder`,
 `CircleBorder`, `RoundedRectangleBorder`, and `StadiumBorder`. A single
-`lerp` function cannot have a priori knowledge of all the possible types,
+`lerp` function cannot have a prior knowledge of all the possible types,
```
Comment (sfshaza2) [inline src/resources/inside-flutter.md:589]:
```suggestion
`lerp` function can't anticipate all possible types,
```

File: src/resources/inside-flutter.md [hunk 2]
```diff
@@ -586,7 +586,7 @@ Some interpolatable objects are defined by class hierarchies. For example,
 shapes are represented by the `ShapeBorder` interface, and there exists a
 variety of shapes, including `BeveledRectangleBorder`, `BoxBorder`,
 `CircleBorder`, `RoundedRectangleBorder`, and `StadiumBorder`. A single
-`lerp` function cannot have a priori knowledge of all the possible types,
```
[the remark this answers is not in the document]
Comment (sfshaza2) [reply src/resources/inside-flutter.md:589]: Yeah, it's better to avoid the Latin altogether.

**Review by sfshaza2 — APPROVED** [review]
Thanks for pointing this confusing text out, @satwanjyu! lgtm

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**